exporters/jaeger/include/opentelemetry/exporters/jaeger/jaeger_exporter.h

```diff
@@ -25,10 +26,11 @@ class ThriftSender;
  */
 struct JaegerExporterOptions
 {
-  // The endpoint to export to.
-  std::string server_addr          = "localhost";
-  uint16_t server_port             = 6831;
   TransportFormat transport_format = TransportFormat::kThriftUdpCompact;
+  std::string endpoint             = "localhost";
+  uint16_t server_port             = 6831;
+  // Only applicable when using kThriftHttp transport.
+  std::vector<HttpHeader> headers;
```

Comment:
nit - should we directly use `ext::http::client::Headers` here, we can get rid of `jaeger_http.h`.

Reply:
Switched it to `ext::http::client::Headers`